src/Symfony/Component/Intl/NumberFormatter/NumberFormatter.php

```diff
@@ -331,7 +331,7 @@ public function formatCurrency($value, $currency)
 
         $value = $this->formatNumber($value, $fractionDigits);
 
-        $ret = $symbol.$value;
+        $ret = $symbol.(mb_strlen($symbol, 'UTF-8') > 2 ? "\xc2\xa0" : '').$value;
```

Comment:
There's now a non-breaking space after the currency code (`CRC 100`), but not if the currency has a symbol (`£100`).